src/main/java/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -13,36 +26,107 @@
+            // Indexer
+            public static final int LIFT_BELT_MOTOR = 6;
+            // Flywheel
+            public static final int FLYWHEEL_MOTOR_A = 2;
+            public static final int FLYWHEEL_MOTOR_B = 1;
+            // Shooter
+            public static final int RUN_UP_BELT_MOTOR = 8;
         }
 
-        public static class PWM {
+        public static class PWM { // TODO: CHANGE CONSTS
         }
 
-        public static class CAN {
+        public static class CAN { // TODO: CHANGE CONSTS
+            public static final int HOOD_MOTOR = -1;
+            public static final int HOOD_ENCODER = -1;
         }
 
-        public static class Pneumatics {
+        public static class Pneumatics { // TODO: CHANGE CONSTS
+            public static final PCMPort INTAKE_SOLENOID = new PCMPort(0, -1, -1);
+            public static final PCMPort FLIPPER_SOLENOID = new PCMPort(0, -1, -1);
+            public static final PCMPort SHOOTER_AIM_SOLENOID = new PCMPort(0, -1, -1);
         }
 
-        public static class Digital {
+        public static class Digital { // TODO: CHANGE CONSTS
+            public static final int INDEXER_LIMIT_SWITCH = -1;
+            public static final int HOOD_LOWER_LIMIT_SWITCH = -1;
+            public static final int HOOD_UPPER_LIMIT_SWITCH = -1;
         }
     }
 
     public static class Metrics {
         public static class Chassis {
             public static final double TICKS_PER_REVOLUTION = -1; // TODO: CHANGE CONSTS
-            public static final double DIAMETER_INCHES = -1;
-            public static final double CIRCUMFERENCE_INCHES = Metrics.Chassis.DIAMETER_INCHES * Math.PI;
+            public static final double DIAMETER_METERS = -1;
+            public static final double CIRCUMFERENCE_METERS = Metrics.Chassis.DIAMETER_METERS * Math.PI;
             public static final double TICKS_PER_INCH = Metrics.Chassis.TICKS_PER_REVOLUTION
-                    / Metrics.Chassis.CIRCUMFERENCE_INCHES;
+                    / Metrics.Chassis.CIRCUMFERENCE_METERS;
             public static final double DISTANCE_FRONT_BACK = -1;
             public static final double DISTANCE_SIDE_SIDE = -1;
-            public static final double INCHES_PER_TICK = Metrics.Chassis.CIRCUMFERENCE_INCHES
+            public static final double METERS_PER_TICK = Metrics.Chassis.CIRCUMFERENCE_METERS
                     / Metrics.Chassis.TICKS_PER_REVOLUTION;
         }
+
+        public static class Encoders {
+            public static class TalonEncoders {
+                public static final double TICKS_PER_REVOLUTION = 2048.0;
+                public static final double REVOLUTIONS_PER_TICK = 1 / TICKS_PER_REVOLUTION;
+            }
+
+            public static class CANCoders {
+                public static final double TICKS_PER_REVOLUTION = 4096.0;
+                public static final double REVOLUTIONS_PER_TICK = 1 / TICKS_PER_REVOLUTION;
+            }
+        }
+
+        public static class Hood {
```

Comment:
Where are the other constants, for converting from motor spins to hood movement?